### PR TITLE
Added HR Lead opening #1906

### LINKED ISSUE #1906: Add "Head of Human Resources" job to website
Add the following job opening to the website.

***

#### Job Description
Our company is looking for a Head of Human Resources to join our team. to oversee all personnel-related matters and ensure we use our resources beneficially.

#### Responsibilities
- a. Working with team leads to write job descriptions
- b. Overseeing the application process
- c. Overseeing the interview process
- d. Overseeing payments and accounting 
- e. Documenting standard memo conventions for payments
- f. Assisting team leads with the onboarding process
- g. Oversee creation of the new recruiting team

#### Technology Requirements
- a. Leadership abilities
- b. Familiar with Github
- c. Knowledgable with Gsuite

#### Reports To
- First Name: [Chrisha]
- Discord Username: [MsChrishaph#0709]

### PR DESCRIPTION
**Pull request by avinashupadhya99**
Closes #1906 

Account number - `94cf53921e9c5ddf48dc5589b7cdf111f7acdc93dc3073b94880416d4676b9ae`

### CODE REVIEW

File: src/data/openings.json [hunk 1]
```diff
@@ -87,5 +87,26 @@
...
+      "Overseeing the interview process.",
+      "Overseeing payments and accounting.",
+      "Documenting standard memo conventions for payments.",
+      "Assisting team leads with the onboarding process.",
+      "Oversee creation of the new recruiting team."
+    ],
+    "technologyRequirements": [
+      "Leadership abilities",
+      "Familiar with Github",
+      "Knowledgable with Gsuite"
```
Comment (jamessspanggg) [inline src/data/openings.json:108]: * knowledgeable

File: src/data/openings.json [hunk 2]
```diff
@@ -87,5 +87,26 @@
...
+    "description": "Our company is looking for a Head of Human Resources to join our team. to oversee all personnel-related matters and ensure we use our resources beneficially.",
+    "position": "Head of Human Resources",
+    "responsibilities": [
+      "Working with team leads to write job descriptions.",
+      "Overseeing the application process.",
+      "Overseeing the interview process.",
+      "Overseeing payments and accounting.",
+      "Documenting standard memo conventions for payments.",
+      "Assisting team leads with the onboarding process.",
+      "Oversee creation of the new recruiting team."
```
Comment (jamessspanggg) [inline src/data/openings.json:103]: `Overseeing the creation... ` to standardise with the previous ones

**Review by jamessspanggg — APPROVED** [review]

File: src/data/openings.json (hunk 2 above)
Comment (avinashupadhya99) [reply src/data/openings.json]: Fixed in adf35aa8e7145635cf1d7e25095b1c2db565eb9a

File: src/data/openings.json (hunk 1 above)
Comment (avinashupadhya99) [reply src/data/openings.json]: Fixed in adf35aa8e7145635cf1d7e25095b1c2db565eb9a

**Comment by jamessspanggg** [comment]
hi @avinashupadhya99 can resolve the conflicts?

**Pull request merged**